plugins/functions/time.c

```diff
@@ -127,6 +127,16 @@ func_time_classify_raw(grn_ctx *ctx,
     grn_obj_cast(ctx, interval, &casted_interval, GRN_FALSE);
     interval_raw = GRN_UINT32_VALUE(&casted_interval);
     GRN_OBJ_FIN(ctx, &casted_interval);
+
+    if (interval_raw == 0) {
+      GRN_PLUGIN_ERROR(ctx, GRN_INVALID_ARGUMENT,
+                       "%s(): "
+                       "the second argument must not be zero: "
+                       "<%u>",
+                       function_name,
+                       interval_raw);
```

Comment:
`"<%u>"` and `interval_raw` are meaningless because message says `must not be zero`.
Please remove them or show `interval`. `interval` will be helpful. Non-`0` `interval` value may be zero such as `-0`. (Maybe overflowed case too?)